### DISCUSSION TITLE
Problems coding in Rust

### DISCUSSION THREAD

**Discussion by renato145** [LazyVim/LazyVim#4930] [Q&A]
Is anyone else having issues with rust, I have encountered the following:
1. This keeps spawning in the editor: `notify.error rust-analyzer: -32802: server cancelled the request`
2. Accepting autocompletes sometimes complete with `...`, eg:  `eprintln!(…)`
3. Using <C-n> and <C-p> to navigate autocompletes change the completions.
4. Accepting autocompletes fill something different that the shown option.

**Comment by aniaan** [comment]
see: https://github.com/neovim/neovim/issues/30985
The simplest way is to upgrade neovim to the latest nightly version.
